arenadata/README.md

```diff
@@ -0,0 +1,30 @@
+# Run regression tests in Docker container
+
+1. Build Docker container with GPDB from source directory:
+```
+docker build -t "adb-6.x-dev:latest" -f arenadata/Dockerfile .
```

Comment:
Do we really need to rebuild gpdb container each time? Why not reuse `hub.adsw.io/library/gpdb6_regress:latest` in most cases from our registry? But you may provide a link to the appropriate article from gpdb repo here.